src/SourceBuild/patches/winforms/0001-Don-t-error-when-file-copy-fails-due-to-locking-unti.patch

```diff
@@ -0,0 +1,25 @@
+From a91eef2a8e4c53f4674b24ff162622f3c80c054d Mon Sep 17 00:00:00 2001
+From: Jeremy Koritzinsky <jekoritz@microsoft.com>
+Date: Fri, 9 Feb 2024 10:59:55 -0800
+Subject: [PATCH] Don't error when file copy fails due to locking until all the
+ backoffs have failed
+
```

Comment:
Please add a backport comment here.  It helps knowing when this patch can be removed in the dependency flows.

https://github.com/dotnet/source-build/blob/main/Documentation/patching-guidelines.md#patch-guidelines

Reply:
```suggestion

Backport: https://github.com/dotnet/winforms/pull/10858
```